src/lazy/streaming_raw_reader.rs

```diff
@@ -119,6 +113,10 @@ impl<Encoding: Decoder, Input: IonInput> StreamingRawReader<Encoding, Input> {
         self.read_next(context, /*is_peek=*/ false)
     }
 
+    #[allow(dead_code)]
+    // TODO: The lower-level readers support a 'peek' operation; it isn't currently exposed at higher
+    //      levels because that would require storing ephemeral values that are peeked.
+    //      We should either remove this feature or finish it.
```

Comment:
```suggestion
    // TODO: The lower-level readers support a 'peek' operation; it isn't currently exposed at higher
    //      levels because that would require storing ephemeral values that are peeked.
    //      We should either remove this feature or finish it.
    //      See: https://github.com/amazon-ion/ion-rust/issues/925
```